### DISCUSSION TITLE
Explain "cannot be inherited" for a non-sealed Class.

### DISCUSSION THREAD

**Discussion by WispArchitecture** [dotnet/runtime#63097] [Q&A] [answered]
Some classes (like `ServiceContainer`) say, "This class cannot be inherited." in the docs, but the classes are not sealed. Deriving from them seems to work fine. There are several cases. If this is valid, the docs should offer an explanation for this.

**Comment by Clockwork-Muse** [comment, marked as answer]
They might not be now, but for at least `ServiceContainer` they used to be (looks like it was unsealed in .NetF 2.0, so quite a long time ago).
This would be a documentation bug, then.
